entities.py

```diff
@@ -0,0 +1,44 @@
+"""
+LibrePCB S-expression entities
+"""
+
+
+class Position(object):
+    def __init__(self, x: float, y: float):
+        self.x = x
+        self.y = y
+
+    def to_s_exp(self) -> str:
```

Comment:
Why not use `__str__` directly? That would simplify string formatting.

Regarding indentation, we have an [`indent`](https://github.com/LibrePCB/librepcb-parts-generator/blob/bb1a73de9c6829d02684393f527b84a80c8e688c/common.py#L67) function that currently works with string lists, but we could probably implement the same thing by doing something like this:

```python
def indent(level: int, val: str) -> str:
    prefix = level * ' '
    return prefix + val.replace('\n', prefix + '\n')
```

Reply:
> Why not use __str__ directly? That would simplify string formatting.

The thing is, that I'm not sure if I should implement `to_s_exp` as returning a `list` of `str` or just a `str` with multiple lines. Having just a `str` makes us splitting the `str`s all the time, which may be bad from a performance and more important ergonomics point of view.

Reply:
No, you don't need to split, you can simply replace `\n` with `\n + prefix`.

Reply:
I'd also argue that performance is not really relevant, especially not in the sub-ms range. And the ergonomics are not a problem if indentation is done with a helper function.

Reply:
> No, you don't need to split, you can simply replace \n with \n + prefix.

This will actually indent the next line, so the first line won't be indented. Did you mean `prefix + \n`?

Reply:
Switched to `__str__` we'll figure out the best way to solve the indentation /  prefixing :slightly_smiling_face: